src/cloud/cdn/configure-fastly-customize-cache.md

```diff
@@ -0,0 +1,226 @@
+
+Edge modules allow you to enable specific functionality like CORS headers, Magento Cloud Sitemap rewrites, and to configure integration between your Magento store and other CMSs or backends.
+
+You must turn on the _Enable Fastly Edge modules_ option to access the Edge Modules menu to view, configure, and manage the available Edge modules.
+
+## Configure backends and Origin shielding
+
+Backend settings provide fine tuning for Fastly performance with Origin shielding and timeouts. A _backend_ is a specific location (IP or domain) with configured Origin shield and timeout settings for checking and providing cached content.
+
+_Origin shielding_ routes all requests for your store to a specific Point of Presence (POP). When a request is received, the POP checks for cached content and provides it. If it is not cached, it continues to the Shield POP, then to the Origin server which caches the content. The shields reduce traffic directly to the origin.
+
+The default Fastly VCL code specifies default values for Origin shielding and timeouts for your {{ site.data.var.ece }} sites. We recommend using the default values. In some case, you might need to modify the default values. For example, if you are getting a lot of time to first byte (TTFB) errors, you might need to adjust the _first byte timeout_ value.
+
+ {:.bs-callout-info}
+If you need to integrate additional backends into your site such as a backend to serve blog content from a [Wordpress]({{ site.baseurl }}/cloud/cdn/fastly-vcl-wordpress.html) site, you must customize your Fastly service configuration to add the backend and handle the redirects from your {{ site.data.var.ee }} store to the Wordpress backend. For details, see [Fastly Edge Modules - Other CMS/Backend integration](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/Edge-Modules/EDGE-MODULE-OTHER-CMS-INTEGRATION.md) in the Fastly module documentation.
+
+{:.procedure}
+To review the backend settings configuration:
+
+{% include cloud/admin-ui-login-step.md %}
+
+1. Click **Stores** > Settings > **Configuration** > **Advanced** > **System** and expand **Full Page Cache**.
+
+1. Expand the **Fastly Configuration** section.
+
+1. Expand **Backend settings** and click the gear to check the default backend. A modal opens that shows current settings with options to change them.
+
+   ![Modify the backend]({{ site.baseurl }}/common/images/cloud/cloud_fastly-backend.png){:width="600px"}
+
+1. Select the **Shield** location (or datacenter) closest to your Cloud service region.
+   For example, if your project is hosted on AWS, Staging is on the west coast of the United States (us-west-1), select the `sjc-ca-us` Fastly shield location. This is the POP that provides caching services.
```

Comment:
Need an extra line.
Also, the "For example" sentence reads a little odd to me with a comma after AWS and a comma after US. Maybe check the sentence to make sure it reads okay. I had to stop a few times, because for me it starts with an "If/then" and then it has a "select action"...yet, they all run together in one sentence. I would separate the two with something a little stronger...like the following suggestion:

```suggestion
1. Select the **Shield** location (or datacenter) closest to your Cloud service region.

   For example, if your project is hosted on AWS, then the Staging is on the west coast of the United States (`us-west-1`); therefore, you would select the `sjc-ca-us` Fastly shield location. This is the POP that provides caching services.
```